### PR TITLE
Enhancement of Scrollbar | Issue #166

### LINKED ISSUE #166: [Feature Request] Scrollbar (Webpage Scrollbar) Updation That Matches with The WebApp Theme.
### Is your feature request related to a problem? Please describe.
Yes, a customised scrollbar is the important part of most of the webapps, that enhances the UI and UX.

### Describe the solution you'd like
I Want to customise and enhance the scrollbar of the webapp.
image

---

**Would you like to work on this feature?**  
- [X] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by Shariq2003**
### Description
Fix #166 
image

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -2,6 +2,24 @@
 @tailwind components;
 @tailwind utilities;
 
+::-webkit-scrollbar {
+    width: 12px;
```
Comment (ajay-dhangar) [inline src/css/custom.css]: replace `width: 12px;` with `width: 5px;`

File: src/css/custom.css [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+    background: #2a2a2a;
+}
+
+::-webkit-scrollbar-thumb {
+    background: #444444;
```
Comment (ajay-dhangar) [inline src/css/custom.css]: replace `background: #444444;` with `background: var(--ifm-color-primary);`

File: src/css/custom.css [hunk 3]
```diff
@@ -2,6 +2,24 @@
...
+    border-radius: 6px;
```
Comment (ajay-dhangar) [inline src/css/custom.css]: replace `border-radius: 6px;` with `border-radius: 3px;`

File: src/css/custom.css [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
+}
+
+::-webkit-scrollbar-thumb:hover {
+    background: #666;
```
Comment (ajay-dhangar) [inline src/css/custom.css]: replace `background: #666;` with `background: var(--ifm-color-primary-dark);`

File: src/css/custom.css [hunk 5]
```diff
@@ -2,6 +2,24 @@
...
+}
+
+::-webkit-scrollbar-track {
+    background: #2a2a2a;
```
Comment (ajay-dhangar) [inline src/css/custom.css]: replace `background: #2a2a2a;` with `background: #48df84;`

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Shariq2003** [comment]
@ajay-dhangar please check now

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**